gui/src/mainwindow.cpp

```diff
@@ -1434,9 +1428,10 @@ void MainWindow::boardDiscovered(BoardType *brd)
     // Board discovered, save it
     currentboard = brd;
 
-    // Stack widget changes to hide some info depending on board
+    // GUI changes info depending on board type
     if(brd->boardName() == "NANO33BLE" ||
-       brd->boardName() == "DTQSYS") {
+       brd->boardName() == "DTQSYS" ||
```

Comment:
at some point the brd object or something in the actual header can do this without an if statement -- generally, as you see, the if(type1, or typr2, etc) gets out of hand -- so prob could just have generic setVisible(brd->some vis field)

Reply:
Sounds like a plan, I think having the board tell the GUI what features are programmed into the firmware rather than having any board types in the GUI at all would be ideal.